code/game/objects/effects/contraband.dm

```diff
@@ -615,4 +615,10 @@
 	desc = "A poster advertising the escapades of Dick Gumshue, mouse detective. Encouraging crew to bring the might of justice down upon wire saboteurs."
 	icon_state = "poster36_legit"
 
+/obj/structure/sign/poster/official/there_is_no_gas_giant
+	name = "There Is No Gas Giant"
+	desc = "Nanotrasen has issued posters, like this one, to all stations reminding them that rumours of a gas giant are false."
```

Comment:
surely at least one of their stations must orbit a gas giant?

and the way this is phrased, it sounds like they're trying to say that gas giants just don't exist anywhere

Reply:
GAS GIANTS DONT EXIST!!

Reply:
Gas Planet Canon
Gas Planet Real
Don't believe in Syndicate Lies